common/src/main/java/com/wynntils/features/user/map/MinimapFeature.java

```diff
@@ -110,8 +110,11 @@ protected MinimapOverlay() {
                     new GuiScaledOverlaySize(DEFAULT_SIZE, DEFAULT_SIZE));
         }
 
+        // FIXME: This is the only overlay not to use buffer sources for rendering. This is due to `createMask`
```

Comment:
Sounds a bit disappointing..? Is not the minimap the most important of all to have the buffered rendering? Or am I missing something?

Reply:
Pois are rendered with buffers, the actual map part and texts, borders etc are small-ish

Reply:
Ah, ok.